docs/src/modules/components/CodeCopyButton.tsx

```diff
@@ -22,6 +23,7 @@ export default function CodeCopyButton(props: CodeCopyButtonProps) {
       className="MuiCode-copy"
       onClick={async (event) => {
         event.stopPropagation();
```

Comment:
Why do we stop propagation?

Reply:
No Idea, it's been here since the creation of editable code in #34454

@bharatkashyap do you remember why you introduced it?

Reply:
In the unknown, it could be nice to drop it, for the sake of learning faster. If it's really needed, we can add a comment. I don't see why it would help in any way. `stopPropagation()` are usually flags, most are harmful: https://css-tricks.com/dangers-stopping-event-propagation/.